revalidate/index.d.ts

```diff
@@ -0,0 +1,69 @@
+// Type definitions for revalidate 1.1.0
```

Comment:
Please add a [tslint.json](https://github.com/DefinitelyTyped/DefinitelyTyped#lint) file, this should catch issues like the invalid version tag above.